package.json

```diff
@@ -99,21 +99,19 @@
         "jest": "^26.4.2",
         "jest-config": "^26.4.2",
         "jest-environment-jsdom": "^26.6.2",
-        "lint-staged": "^11.1.1",
-        "prettier": "^2.3.2",
         "rimraf": "^3.0.2",
         "rollup-plugin-typescript2": "^0.27.2",
         "run-with-testrpc": "0.3.1",
         "semantic-release": "17.3.0",
         "ts-jest": "26.3.0",
         "typechain": "^2.0.0",
-        "typedoc": "^0.19.2",
-        "typedoc-plugin-markdown": "~3.0.11",
+        "typedoc": "^0.21.5",
+        "typedoc-plugin-markdown": "~3.10.4",
         "typescript": "^4.0.2"
     },
     "husky": {
         "hooks": {
-            "pre-commit": "lint-staged && rm -rf docs/api && typedoc --plugin typedoc-plugin-markdown --hideBreadcrumbs true && git add -A docs/api/"
```

Comment:
just curious as to why we are removing `lint-staged` here?

Reply:
its moving as a separate command with additional options below the husky option

https://github.com/energywebfoundation/iam-client-lib/pull/226/commits/f82c8b07e9c7e10a50e7e6d9d35146cf367dec96#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R117

Reply:
resolved https://github.com/energywebfoundation/iam-client-lib/pull/226/commits/2c4d6d2e0308ebc32f42574a5a620685b000f824